deps/curl-7.74.0/lib/vtls/sf_ocsp.c

```diff
@@ -2175,13 +2176,13 @@ SF_PUBLIC(CURLcode) checkCertOCSP(struct connectdata *conn, STACK_OF(X509) *ch,
     ocsp_fail_open = DISABLED;
   }
 
-  sf_otd_set_insecure_mode(1, ocsp_log_data);
+  sf_otd_set_insecure_mode(1, &ocsp_log_data);

```

Comment:
This should be set to 0. This fix is unrelated to your change but let's just fix it. @SimbaGithub